lib/pulselistener.js

```diff
@@ -65,12 +83,21 @@ var PulseConnection = function(options) {
     namespace:          options.username || ''
   });
 
-  if (!options.connectionString) {
-    options.connectionString = buildPulseConnectionString(options);
-  } else {
+  this.isTaskclusterPulse = false;
```

Comment:
It's actually not private at the moment. See [L330](https://github.com/taskcluster/taskcluster-client/pull/64/files#diff-bdeec392e02b8fc5c8a9ac0b62f3b398R330), where `PulseListener` checks the `isTaskclusterPulse` property of `PulseConnection`. Is it better to also create this property in `PulseListener` and make both private?